src/content/docs/ai-monitoring/intro-to-ai-monitoring.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: 'Introduction to AI monitoring'
+metaDescription: 'AI monitoring lets you observe the AI-layer of your tech stack, giving you a holistic overview of the health and performance of your AI-powered app.'
+freshnessValidatedDate: never
+---
+
+import aiTraceViewIntroPage from 'images/ai_screenshot-full_trace-view-intro-page.webp'
+
+import aiAIResponsesOverview from 'images/ai_screenshot-full_AI-responses-overview.webp'
+
+When New Relic talks about AI, we're referring to the layer of your environment that uses a large language model (LLM) to generate a response when it receives an end user prompt. Your app contains an AI assistant, but behind the AI are libraries, cost implications, vector databases, and new processes that can be monitored.   
```

Comment:
that can be monitored > that you can monitor (passive fix).

Also I feel like the section under the picture should be moved up to the above the picture to provide further context for what the picture is showing. I think you can just add something short beneath the picture describing what the goal of the intro doc is (what the user should expect to learn about if they finish reading the doc). Normally I try to steer away from "doc summaries" in the beginning but I think intro docs are an exception.